run-style-checks.sh

```diff
@@ -7,6 +7,6 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 pydocstyle invenio_app_ils tests docs && \
-isort -rc -c -df && \
+isort -rc -c --df && \
```

Comment:
```suggestion
isort --check-only --diff && \
```
suggested [here](https://github.com/inveniosoftware/cookiecutter-invenio-instance/pull/238)